CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ add_library(ViveLoaderLib STATIC
     VRSettings.h)
 target_link_libraries(ViveLoaderLib
     PUBLIC
-    OpenVRDriver osvr::osvrUtil linkable_into_dll
+    OpenVRDriver osvr::osvrUtil linkable_into_dll ${CMAKE_DL_LIBS}
```

Comment:
I'd prefer for this to be conditional on `if(NOT WIN32)`, if you don't mind. It might also be able to be a `PRIVATE` library dependency, rather than public, not sure.